### PR TITLE
release(web): Centreon Web 20.04.6 RN

### PR DESCRIPTION
**Pull request by lpinsivy**
Close #270

### CODE REVIEW

File: en/releases/centreon-core.md [hunk 1]
```diff
@@ -18,6 +18,64 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web release notes
 
+### 20.04.6
+
+> Newly shared views do not break widget preferences.
+> But, if you have already broken widget preferences for users who add a shared view, you'll need to :
+> * Login centreon web with the user who share the custom view
```
Comment (thiuyendang) [inline en/releases/centreon-core.md:25]: ...who shares

File: en/releases/centreon-core.md [hunk 2]
```diff
@@ -18,6 +18,64 @@ If you have feature requests or want to report a bug, please go to our
...
+* [Widgets] Parameters are deleted when importing/deleting/importing a custom view
+
+#### Security fixes
+
+* [API] Information Disclosure in centreon_wiki internal API
+* [API] ]Cross-site Scripting (XSS) Reflected in centreon_wiki internal API
+* [Administration] Horizontal privilege escalation / session takeover
+* [Configuration] Cross Site Scripting in widget rename
+* [Configuration] RCE in SNMP trap import
+* [Configuration] Vulnérabilités d’injections SQL in "Configuration > Host categories"
```
Comment (thiuyendang) [inline en/releases/centreon-core.md:73]: Injection vulnerabilities ?

File: en/releases/centreon-core.md [hunk 3]
```diff
@@ -18,6 +18,64 @@ If you have feature requests or want to report a bug, please go to our
...
+* [Configuration] Vulnérabilités d’injections SQL in "Configuration > Service Groups"
```
Comment (thiuyendang) [inline en/releases/centreon-core.md:74]: Injection vulnerabilities ?

File: en/releases/centreon-core.md [hunk 4]
```diff
@@ -18,6 +18,64 @@ If you have feature requests or want to report a bug, please go to our
...
+* [Configuration] Vulnérabilités d’injections SQL in "Configuration > Service categories"
```
Comment (thiuyendang) [inline en/releases/centreon-core.md:75]: Injection vulnerabilities ?

**Review by thiuyendang — CHANGES_REQUESTED** [review]

File: en/releases/centreon-core.md [hunk 5]
```diff
@@ -18,6 +18,66 @@ If you have feature requests or want to report a bug, please go to our
...
+>
+> * Login centreon web with the user who shares the custom view
+> * Switch to the custom view with broken preferences for other users
+> * Click on "Share view", and then click on "Share"
+>
+> This will restore preferences for other users
+
+#### Enhancements
+
+* [Event View] Be able to create & save filters
```
Comment (kduret) [inline en/releases/centreon-core.md:35]: to be consistent with line 39
```suggestion
* [Event View] Be able to create and save filters
```

File: fr/releases/centreon-core.md [hunk 6]
```diff
@@ -18,6 +18,66 @@ If you have feature requests or want to report a bug, please go to our
...
+>
+> * Login centreon web with the user who shares the custom view
+> * Switch to the custom view with broken preferences for other users
+> * Click on "Share view", and then click on "Share"
+>
+> This will restore preferences for other users
+
+#### Enhancements
+
+* [Event View] Be able to create & save filters
```
Comment (kduret) [inline fr/releases/centreon-core.md:35]: to be consistent with line 39
```suggestion
* [Event View] Be able to create and save filters
```

Comment (thiuyendang) [reply fr/releases/centreon-core.md:35]: some new fixes have been added in the version and not tested yet, including one new today so I will check the list when available

**Comment by cgagnaire** [comment]
Can you change the base branch pls ?

File: en/releases/centreon-core.md [hunk 7]
```diff
@@ -18,6 +18,66 @@ If you have feature requests or want to report a bug, please go to our
...
+* [Event View] When you empty the ack windows, it starts an infinite load
+* [Eventlog] Acknowledged alerts status show "OK" but it's wrong
+* [Graphs][legacy pages] 1000/1024 graph template ignored
+* [Internal/API/Chart] Centreon db configuration name hard coded
+* [Monitoring] Status output not correctly displayed with chinese characters
+* [Remote-Server] incorrect url to contact Centreon Central Server
+* [UI] Incoherent paging information display
+* [Widgets] Can't change position of widgets
+* [Widgets] Parameters are deleted when importing/deleting/importing a custom view
+
```
Comment (thiuyendang) [inline en/releases/centreon-core.md:67]: Missing mention to this ticket https://centreon.atlassian.net/browse/MON-6095
This one https://centreon.atlassian.net/browse/MON-6203 has been added too but no need to talk about it as the timelime is added in this version so no user could notice the issue before.

**Review by thiuyendang — CHANGES_REQUESTED** [review]

File: fr/releases/centreon-core.md (hunk 6 above)
Comment (thiuyendang) [reply fr/releases/centreon-core.md:35]: this RN must be cherry-picked on branch 20.04.x of the documentation for it to be released in the 20.04 documentation

**Review by kduret — APPROVED** [review]

**Review by cgagnaire — APPROVED** [review]

**Review by thiuyendang — APPROVED** [review]

**Pull request merged**